content-script.js

```diff
@@ -0,0 +1,482 @@
+// Content script for GitHub Draft Issues extension
+(function() {
+    'use strict';
+
+    // Draft storage manager
+    const DraftManager = {
```

Comment:
I'd prefer that draft management be built into the GitHub UI. For example on `/issues`, there would be a section above the issue search results that would show the user's issue drafts for that repository.